README.md

```diff
@@ -67,13 +67,17 @@ Place.create_indexes
 Points
 ------
 
+This gem defines a specific Point class under the Mongoid::Geospatial namespace. Make sure to use `type: ::Mongoid::Geospatial::Point` to avoid name errors or collisions with other Point class you might already have defined `NameError`s
```

Comment:
I would back-tick quote `Point`, change to "classes" and add a period at the end of this.